brayns/common/geometry/SDFBezier.h

```diff
@@ -0,0 +1,141 @@
+/* Copyright (c) 2015-2018, EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ * Responsible Author: Sebastien Speierer <sebastien.speierer@epfl.ch>
```

Comment:
```suggestion
```

Reply:
Why should I remove those?

Reply:
We do not really enforce these responsibility anyway, and if you want to find out who wrote some code then using git blame is much better. It is also common to copy paste the header and forget to update the author :)

Reply:
So should I remove those everywhere?

Reply:
I guess you could, but then I think it should be in a separate PR.